modules/functions/src/main/java/org/apache/tamaya/functions/PropertySourceFunctions.java

```diff
@@ -198,7 +198,7 @@ public static boolean isKeyInSections(String key, String[] sectionKeys) {
             int index = section.lastIndexOf('.');
             if (index < 0 && section.isEmpty()) {
                 transitiveAreas.add("<root>");
-            } if (index < 0) {
+            } else if (index < 0) {
```

Comment:
I would point out this modification as an actual change in logical flow. From the code, it appeared that the intention here was to include `else if` rather than just `if`, but if that is incorrect, I can revert this change (putting the `if` on a new line). An `else if` here prevents empty strings from being added to the `transitiveAreas` Set; otherwise, an empty `section` String would result in both `"<root>"` and `""` being added to this Set.

Reply:
To my mind the change makes sense :+1: